config/environment.rb

```diff
@@ -1,5 +1,20 @@
 # Load the Rails application.
 require_relative 'application'
+require_relative 'initializers/rollbar'
 
-# Initialize the Rails application.
-Rails.application.initialize!
+notify = lambda do |e|
```

Comment:
This feels a little abnormal to me: is this a standard rollbar thing? Is there a reason we're adding this?

Reply:
It is: https://docs.rollbar.com/docs/rails#rails-booting-process but added it because if there was something wrong, being able to get the error logs would be rather useful!

Can remove, but more thinking of if there was an issue, it's better to know (even if it becomes obvious by checking fly.io logs, having it external is also good)